pkg/datapath/linux/modules/modules.go

```diff
@@ -50,3 +45,11 @@ func (m *ModulesManager) FindOrLoadModules(expectedNames ...string) error {
 	}
 	return nil
 }
+
+func newManager(lc hive.Lifecycle) *Manager {
+	mm := Manager{}
+
+	lc.Append(hive.Hook{OnStart: mm.Start})
```

Comment:
Hmm, I'm not sure what this does. `mm` is `Manager` and not `*Manager`, so there might be a copy happening here. Please do `m := &Manager{}` instead. And please call it `m` instead of `mm` to match method receivers.

Reply:
Good catch.